spec/presenters/fulfilment_error_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+RSpec.describe FulfilmentError do
+  subject {
+    described_class.new(:foobar, message: 'Foobar message', error: :blank, change_path: 'steps/foo/bar')
+  }
+
+  describe '#to_partial_path' do
+    it 'returns the correct partial path' do
+      expect(subject.to_partial_path).to eq('steps/application/check_your_answers/fulfilment_error')
+    end
+  end
+
+  describe 'attributes accessors' do
```

Comment:
I think you maybe have mistakenly used `describe` where you should be using `context` and vice versa

Reply:
Yep, in this scenario changing describe with context and vice versa will probably read better. Will push an update!